src/containers/Login/LoginFailure.tsx

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (C) 2021 -present, NDLA
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { injectT, tType } from '@ndla/i18n';
+
+export const LoginFailure = ({ t }: tType) => {
+  const userNotRegistered = false;
+
+  return (
+    <div>
+      <h2>{t('loginFailure.errorMessage')}</h2>
+      {userNotRegistered && <p>{t('loginFailure.userNotRegistered')}</p>}
```

Comment:
Denne her vil aldri vises, for du definerer over at `userNotRegistered` er false. Er det noe logikk som mangler her for å sjekke om brukeren faktisk er registered?